### ISSUE TITLE
categorical_crossentropy implementation

### ISSUE THREAD

**Issue by ericpresas** [fchollet/keras#6784]
Anybody knows how is implemented the categorical cross entropy loss function?

![image](https://cloud.githubusercontent.com/assets/17392065/26539131/211fefb6-444b-11e7-81ec-b57a658ae0c9.png)

I mean if it is implemented with a summation or just taking the maximum value over the original labels (one hot encoding).

Thanks!!
